ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/EngineJsonRpcService.java

```diff
@@ -537,8 +561,8 @@ private void applyTlsConfig(final HttpServerOptions httpServerOptions) {
       try {
         httpServerOptions
             .setSsl(true)
-            .setPfxKeyCertOptions(
-                new PfxOptions()
```

Comment:
are these equivalent

Reply:
@jflo or @diega do we have any known TLS setups that we can get to test this doesn't break anything?